src/components/MergeInfographic.tsx

```diff
@@ -111,20 +84,55 @@ const MergeInfographic: React.FC<IProps> = ({ className }) => {
     translateMessageId(id, intl)
 
   return (
-    <Container
+    <VStack
       className={className}
       role="img"
       aria-label={translateMessageId(
         "page-upgrades-merge-infographic-alt-text",
         intl
       )}
+      position="relative"
+      width="100%"
+      sx={{
+        isolation: "isolate",
+        aspectRatio: `${25 / 11}`,
+      }}
     >
-      <ExecutionLayer aria-hidden="true">
+      <Box
+        position="absolute"
+        top="44%"
```

Comment:
`top` should have the value of `40%` here.

Reply:
That is weird. You are correct, in the code we have `40%` but in prod, inspecting the element using the dev tools, it says `44%` 🤷🏼 whatever...both values work fine at the end IMO.